.reuse/dep5

```diff
@@ -9,3 +9,7 @@ License: Apache-2.0
 Files: elixir/edgehog_device_forwarder_proto/lib/edgehog_device_forwarder_proto/**/*.ex
 Copyright: 2023 SECO Mind Srl
 License: Apache-2.0
+
+Files: CHANGELOG.md
+Copyright: 2024 SECO Mind Srl
+License: Apache-2.0
```

Comment:
I'd rather have a `CHANGELOG.md.license` instead of a dep5 entry

Reply:
Also our markdown files are usually licensed under some creative commons instead of Apache